tests/unit/twitter.js

```diff
@@ -4,13 +4,14 @@
   var deps = [
     'assert',
     'nock',
+    'underscore',
```

Comment:
lodash is actually like low-dash, its kinda the same as underscore, just a different library.